packages/ui/Scrollable/Scrollable.js

```diff
@@ -182,6 +182,34 @@ class ScrollableBase extends Component {
 		 */
 		onMouseDown: PropTypes.func,
 
+		/**
+		 * Called when reached to an edge.
+		 * Passes `orientation`, `position`, and `reached`.
```

Comment:
* Passes `'orientation'`, `'position'`, and `'reached'`.

Reply:
Nope. they are not string values but names of property.

Reply:
Oh, I missed. Good!!